decidim-initiatives/app/cells/decidim/initiatives/initiative_metadata_g_cell.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Initiatives
+    # This cell renders the assembly metadata for g card
+    class InitiativeMetadataGCell < Decidim::CardMetadataCell
+      include Cell::ViewModel::Partial
+      include Decidim::Initiatives::InitiativeHelper
+
+      alias current_initiative resource
+      alias initiative resource
+
+      def items
+        ["test"]
```

Comment:
@jorgeatgu is this a pending list? Something for @entantoencuanto ?

Reply:
It was a test, fixed.